interop-cats/jvm/src/main/scala/scalaz/zio/interop/catsjvm.scala

```diff
@@ -27,18 +27,34 @@ import scala.concurrent.duration.{ FiniteDuration, NANOSECONDS, TimeUnit }
 
 abstract class CatsPlatform extends CatsInstances {
   val console = interop.console.cats
+
+  object implicits {
+    implicit def ioTimer[E]: effect.Timer[IO[E, ?]] =
+      new effect.Timer[IO[E, ?]] {
+        override def clock: effect.Clock[IO[E, ?]] = new effect.Clock[IO[E, ?]] {
+          override def monotonic(unit: TimeUnit): IO[E, Long] =
+            zioClock.nanoTime.map(unit.convert(_, NANOSECONDS)).provide(Clock.Live)
```

Comment:
You can just use `Clock.Live.nanoTime` (etc.) directly here and below, that way you avoid the whole `provide` thing. It is shorter and will be faster.